exercises/practice/alphametics/example.rkt

```diff
@@ -0,0 +1,6 @@
+#lang racket
+
+(provide alphametics-for)
+
+(define (alphametics-for puzzle)
+  (error "Not implemented yet"))
```

Comment:
The purpose of the example file is to demonstrate that it is possible to write a solution that passes all the tests. 
The current example file won't, hence the build failing: https://github.com/exercism/racket/actions/runs/3993853554/jobs/6850940870

The build is also failing because it expects the `example.rkt` file to be located in the `.meta` directory. Could you move it there?